setup.cfg

```diff
@@ -0,0 +1,12 @@
+[flake8]
+max-line-length = 80
+ignore =
+  C901 # object names too complex
+  E111, E114 # four-space indents
+  E121 # line continuations
+  W503, W504 # line breaks around binary operators
+max-complexity = 18
+select = B,C,F,W,T4,B9
```

Comment:
What do `B,C,F,...` stand for?

Reply:
It's basically different classes of errors, so that we only select for the ones that are applicable to google style. These ones are from jax, and seem to be fairly common in other projects as well